src/site/shared/containers/ItemNav/index.scss

```diff
@@ -153,9 +153,10 @@ $tab-padding: 8px;
             border-color: transparent;
 
             cursor: pointer;
-
+            pointer-events: none;
+            
             transition: 0.5s;
-
+            
```

Comment:
nit: This indentation is still present

Reply:
Maybe I didn't understand from the way github shows it. Is it saying line 159 should NOT have an indent?

Reply:
It is saying that this pr is adding the highlighted indentation. Previously, this was an empty line, now in this pr it is a line with some whitespace in it.

Reply:
Alright, thanks! still a bit new to how GitHub shows changes